aiohttp/client.py

```diff
@@ -97,6 +98,11 @@
 from .tracing import Trace, TraceConfig
 from .typedefs import JSONEncoder, LooseCookies, LooseHeaders, StrOrURL
 
+if sys.version_info >= (3, 11):
+    from typing import Unpack
+else:
+    from typing_extensions import Unpack
```

Comment:
Or about if `typing` or `typing_extensions` are installed?


if sys.version_info >= (3, 11):
    from typing import Unpack
    RequestArgs = Unpack[_RequestOptions]
else:
    try:
       from typing_extensions import Unpack
       RequestArgs = Unpack[_RequestOptions]
    else:
        RequestArgs = Any

Reply:
Personally for me, I would prefer this, as I maintain 3.8+ projects that I have typing-extensions installed for.

Reply:
So at least I can take advantage of things even if the min version isn't 3.11, while keeping it optional.

Reply:
I guess I'm OK with that.

Reply:
This ends up being problematic, as it's invalid for type-checkers (or at least Pyright) to preassign an `Unpack`. It must be used with key-word argument packing or else it's invalid and leads to a unknnown.

Reply:
This doesn't end up working, since it's not valid to pre-assign `Unpack`, or at least in Pyright. You have to use it immediately after key-word unpacking. 

This is also invalid:

```python
if TYPE_CHECKING:
    if sys.version_info >= (3, 11):
        from typing import Unpack
    else:
        try:
            from typing_extensions import Unpack

            Unpack = Unpack
        except ImportError:
            Unpack = Any
```

As you'll just get a `Variable not allowed in type expression` warning.

Reply:
@Dreamsorcerer Wanted to follow-up here, I think it's going to be tricky to add things in without adding typing-extensions back.

Reply:
Can a single overload be used to replace the function signature?
I don't have time to try right now, but along the lines of:
```
if sys.version_info >= (3, 11):
    @overload
    def get(self, url, kwargs: Unpack[...]):
        ...
def get(self, url, kwargs: Any):
```

Reply:
Good idea, let me give that a try.

Reply:
Late to the party, but I thought I'd mention that type checkers assume `typing_extensions` is part of the stdlib for ease of use. Therefore, limiting this 3.11+ is unnecessary, and you can import `Unpack` inside the type checking block from `typing_extensions`.